metainfo.patch

```diff
@@ -0,0 +1,48 @@
+     <summary>Preview all kinds of files</summary>
+     <description>
+-        <p>PreviewQt is a very simple application for <b>previewing a wide range of files</b>. Its main aim is on <b>speed</b>, striving to provide a preview as instantaneously as possible. It achieves this by being <b>very lightweight</b> and by its ability to sit in the system tray, ready to show whatever is thrown at it.</p>
++        <p>PreviewQt is a very simple application for previewing a wide range of files. Its main aim is on speed, striving to provide a preview as instantaneously as possible. It achieves this by being very lightweight and by its ability to sit in the system tray, ready to show whatever is thrown at it.</p>
+         <p>Supported file formats include:</p>
+         <ul>
+-            <li>More than <b>140 image formats</b> - a list of all possibly supported image formats [can be found here](https://photoqt.org/formats)</li>
+-            <li>All of the common <b>video formats</b></li>
+-            <li><b>Photo spheres</b> and <b>360 degree panoramas</b> (that use equirectangular projections)</li>
+-            <li><b>Motion photos</b>, <b>Micro videos</b>, and <b>Apple live photos</b></li>
+-            <li><b>PDF documents</b> including page navigation</li>
+-            <li><b>Archives</b> (zip, tar.gz, rar, 7-zip) with the ability to explore its content</li>
+-            <li><b>Comic Books</b> (cbz, cbt, cbr, cb7) including page navigation</li>
++            <li>More than 140 image formats</li>
++            <li>All of the common video formats</li>
++            <li>Photo spheres and 360 degree panoramas (that use equirectangular projections)</li>
++            <li>Motion photos, Micro videos, and Apple live photos</li>
++            <li>PDF documents including page navigation</li>
++            <li>Archives (zip, tar.gz, rar, 7-zip) with the ability to explore its content</li>
++            <li>Comic Books (cbz, cbt, cbr, cb7) including page navigation</li>
+         </ul>
+         <p>It is also possible to set up an external application for the categories of files. With a shortcut or button it is possible to pass a previewed file on to the provided external application for further processing.</p>
+-        <p>If you like the overall concept of PreviewQt but would like to have more features, check out the fully featured image viewer PhotoQt: https://photoqt.org</p>
++        <p>If you like the overall concept of PreviewQt but would like to have more features, check out the fully featured image viewer PhotoQt.</p>
+     </description>
+     <screenshots>
+         <screenshot type="default">
+@@ -42,8 +42,8 @@
+             <caption>The integrated help text.</caption>
+         </screenshot>
+     </screenshots>
+-    <launchable type="desktop-id">org.photoqt.PreviewQt.desktop</launchable>
+-    <developer id="lukas.photoqt">
++    <launchable type="desktop-id">org.previewqt.PreviewQt.desktop</launchable>
++    <developer id="lukas.previewqt">
```

Comment:
Should be reverse domain name `org.previewqt`

Reply:
previewqt.org is indeed the domain name for PreviewQt. It currently redirest to https://photoqt.org/previewqt but it will have its own website by the weekend.